build-conflicts.php

```diff
@@ -131,6 +135,62 @@ static function ($errorCode, $message = '', $file = '', $line = 0) : void {
         ));
     };
 
+    /** @return Advisory[] */
+    $getGitHubAdvisories = static function () use ($token) : array {
+        $query = <<<QUERY
+        {
+            securityVulnerabilities(ecosystem: COMPOSER, first: 100 %s) {
+                edges {
+                    cursor
+                    node {
+                        vulnerableVersionRange
+                        package {
+                            name
+                        }
+                    }
+                }
+            }
+        }
+        QUERY;
+
+        $vulnerabilities = [];
+
+        do {
+            if (isset($data) && count($data) !== 0) {
+                $after = sprintf(', after: "%s"', $data[count($data) - 1]['cursor']);
+            }
+
+            $context = stream_context_create([
```

Comment:
Let's use a PSR-18 HTTP client here - that should make it a bit easier to stub this